sms_939/wizards/sms_sender_wizard.py

```diff
@@ -84,20 +85,24 @@ def send_sms_partner(self):
         if not self.partner_id or not self.partner_id.mobile:
             return False
         if self.send_sms(self.partner_id.mobile.replace(u'\xa0', u'')):
-            self.partner_id.message_post(body=self.text,
-                                         subject=self.subject,
-                                         partner_ids=self.partner_id,
-                                         type='comment')
+            self.env['sms_sponsorship'].create({
+                'partner_id': self.partner_id.id,
+                'text': self.text,
+                'subject': self.subject,
+                'date': datetime.today()
```

Comment:
`fields.Datetime.now()` will also remove the added import of the datetime library.

Reply:
Done.